accessibility-checker-engine/help/HAAC_Video_HasNoTrack.mdx

```diff
@@ -21,13 +21,14 @@ Media embedded with a `<video>` element must have a `<track>` element to provide
 
 ### Why is this important?
 
-Text tracks contain a textual transcription of either audio (captions) or visual (visual/video description) aspects of video or audio content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues, or information about characters, location, actions and scene changes. Providing text alternatives makes information available that some people would not have access to otherwise.
+Text tracks or open captions contain a textual transcription of audio (captions) aspects of video content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues. Providing text alternatives makes information available that some people would not have access to otherwise.
```

Comment:
```suggestion
Text tracks or open captions contain a textual transcription of audio aspects of video content in a synchronized media. This may include the dialog, sound effects, and relevant musical cues. Providing text alternatives (captions) makes information available that some people would not have access to otherwise.
```